app/javascript/components/pages/FateIndex.js

```diff
@@ -1,13 +1,39 @@
-import React,{Component} from 'react'
+import React, { Component } from "react";
+import { Col, Card, CardSubtitle } from "reactstrap";
 
-class FateIndex extends Component{
-    render() {
-
-        return(
+class FateIndex extends Component {
+  //this page still needs work
+  render() {
+            console.log(this.props.events)
+    return (
+      <>
+      <div>
+          <h2>This is Fate Index Page Test1</h2>         
+            {this.props.events && this.props.events.map(event => {
+              return(
+                <p key={event.id}>
+                  {event.description}
+                </p>            
+          )})}
+      </div>
+      </>
+    );
+  }
+}
+export default FateIndex;
 
-            <h1>This is the FateIndex page</h1>
 
-        )
-    }
-}
-export default FateIndex
+          {/* <br />
```

Comment:
Shouldn't commit commented blocks of code like this... It's not necessary at all. If you wanna keep something around you can commit it so you've got a record in git should you need to go back.

Reply:
Fixed